src/field/index.js

```diff
@@ -94,10 +92,14 @@ class Field {
         )}`;
 
         const field = this._getInitMeta(name);
-        const defaultValue =
-            typeof initValue !== 'undefined'
-                ? initValue
-                : originalProps[defaultValueName];
+        let defaultValue;
+        if (typeof initValue !== 'undefined') {
+            defaultValue = initValue;
+        } else if (originalProps[defaultValueName]) {
+            defaultValue = originalProps[defaultValueName];
+        } else {
+            defaultValue = get(this.initValues, name);
```

Comment:
If no other values to set as default, look at the `initValues` set in constructor.